Plugins/Libs/CAIR_v2.19/main.cpp

```diff
@@ -164,7 +164,7 @@ void Print_Usage()
 	cout << "      Removal: 5" << endl;
 	cout << "      CAIR_HD: 6" << endl;
 	cout << "      Default: CAIR" << endl;
-	cout << "  -C <convoluton_type>" << endl;
+	cout << "  -C <convolution_type>" << endl;
```

Comment:
This might be a third party library, in which case, it might be best for me to send the fix upstream -- if someone identifies the upstream, I'll see about doing that.

Reply:
I think the upstream is no longer active, so there's no real place to send it.